pandas/core/series.py

```diff
@@ -3231,7 +3234,7 @@ def sort_values(
 
         # GH 35922. Make sorting stable by leveraging nargsort
         values_to_sort = ensure_key_mapped(self, key)._values if key else self._values
-        sorted_index = nargsort(values_to_sort, kind, ascending, na_position)
+        sorted_index = nargsort(values_to_sort, kind, bool(ascending), na_position)
```

Comment:
why is this needed?

Reply:
I did that in order to match the expected type for ``ascending`` in ``nargsort`` function.
It must be ``bool`` as per the function signature.
However, at this particular point ``ascending`` can be an integer as well (previously it was allowed).